code/go/0chain.net/miner/m_handler_integration_tests.go

```diff
@@ -252,5 +260,35 @@ func blockWithValidTicketsForOldRound(r *http.Request) (*block.Block, error) {
 // VRFShareHandler - handle the vrf share.
 func VRFShareHandler(ctx context.Context, entity datastore.Entity) (
 	interface{}, error) {
+	cfg := crpc.Client().State().CollectVerificationTicketsWhenMissedVRF
+
+	vrfs, ok := entity.(*round.VRFShare)
+	if !ok {
+		log.Panicf("unexpected type")
+	}
+
+	if cfg != nil && cfg.Miner == node.Self.ID && int64(cfg.Round) == vrfs.Round {
+		logging.Logger.Debug("Skipping VRF share handling")
+		return nil, nil
+	}
+
 	return vrfShareHandler(ctx, entity)
 }
+
+// NotarizedBlockHandler - handles a notarized block.
+func NotarizedBlockHandler(ctx context.Context, entity datastore.Entity) (
+	interface{}, error) {
+	var nb, ok = entity.(*block.Block)
+	if !ok {
+		log.Panicf("unexpected type")
+	}
+
+	cfg := crpc.Client().State().CollectVerificationTicketsWhenMissedVRF
+
+	if cfg != nil && cfg.Miner == node.Self.ID && int64(cfg.Round) == nb.Round {
```

Comment:
we should skip notarizations not only for current round but for all rounds bigger than current. the idea here is to be sure that tested miner could finalize round by itself. strictly, we wan't to be sure that miner did sync for block for which it doesn't have RRS locally, but has all verification tickets

Reply:
resolved in a46ddfdc57f9b9bd3635a66db6f3cda087da6ce5

Reply:
@fabioDMFerreira should it be `int64(cfg.Round) >= nb.Round`?

Reply:
@dabasov should we ignore all notarizations after a specific round?

Reply:
fixed by [fc11135](https://github.com/0chain/0chain/pull/1905/commits/fc11135b79e64257b3acd2e772a71cdd3f364f9f)